heartbeat/docker-compose

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
-# Version: 1.1.2
-# Date: 2020-06-24
+
+# Version: 1.1.1
```

Comment:
You should increase the version nr, not decrease it.

Reply:
fixed